<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CJoinMergeTextWidget</class>
 <widget class="QWidget" name="CJoinMergeTextWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Join/Merge Text (Line by Line)</string>
  </property>
  <widget class="QLabel" name="m_pLabelSelectedFile1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pLineEditSaveAs">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>560</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_pRadioBtnUnix">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>560</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Unix</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnBrowse1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_pRadioBtnDos">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>560</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Dos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnSaveAs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SAVE AS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnLoadFile1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>LOAD FILE</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelInpu1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Input 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnBrowse2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnLoadFile2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>LOAD FILE</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelInpu2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Input 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelSelectedFile2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>90</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelPrefix">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Prefix</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pLineEditPrefix">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_pTextEditInput1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>201</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelDelimiter">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Delimiter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pLineEditDelimiter">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_pTextEditInput2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>190</y>
     <width>201</width>
     <height>120</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnMergeLines">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>MERGE LINES</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelFillVoids">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Fille voids with</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pLineEditFillVoids">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLabelJoinSets">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Join sets with</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pLineEditJoinSets">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>350</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnClearAllOutputText">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>33</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnSelAllOutputText">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>33</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtnClearInput">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>CLEAR INPUT</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_pTextEditOutput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>780</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
